go.sum

```diff
@@ -1,2 +1,25 @@
+github.com/mmcloughlin/avo v0.0.0-20200425082757-f891fe8d9749 h1:4FVJDva0A+gzC/oxjNyfhRTWloxA/FLu7THXvEYvORk=
```

Comment:
Revert to the original `go.mod` and `go.sum` files.

Reply:
Done.